src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1619,13 +1619,17 @@ GlobeControls.prototype.setCameraTargetPosition = function setCameraTargetPositi
         return player.play(animationZoomCenter).then(() => {
             this.resetControls();
             this.waitSceneLoaded().then(() => {
-                this.updateCameraTransformation();
-                ctrl.targetGeoPosition = null;
+                animatedScale = 0;
+                if (player.isStopped()) {
+                    this.updateCameraTransformation();
+                    ctrl.targetGeoPosition = null;
+                }
             });
         });
     } else {
         ctrl.progress = 1.0;
         quatGlobe.setFromUnitVectors(vFrom, vTo);
+        this.updateCameraTransformation(this.states.MOVE_GLOBE, false);
```

Comment:
MOVE_GLOBE is the default state?

Reply:
Default state is `NONE`, in `setCameraTargetPosition uses `MOVE_GLOBE` to move target's position`